### PR TITLE
plan: fix panic when explain

### PR DESCRIPTION
**Pull request by winoros**
Sql like `explain select * from t order by t.v1 limit 0, 1;` will cause a panic because limit is a property of sort and doesn't has a child. If we call MarshalJSON of *Limit, we will get  a `index out of range` panic

PTAL @hanfei1991 @shenli @zimulala @coocood @XuHuaiyu @tiancaiamao

### CODE REVIEW

**Comment by coocood** [comment]
Can we keep the original format?

File: plan/physical_plans.go [hunk 1]
```diff
@@ -620,19 +620,28 @@ func (p *Sort) Copy() PhysicalPlan {
...
-	limit, err := json.Marshal(p.ExecLimit)
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
 	exprs, err := json.Marshal(p.ByItems)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	limitCount := []byte("null")
+	limitOffset := []byte("null")
```
Comment (hanfei1991) [inline plan/physical_plans.go]: limitOffset is useless, because it will always be 0

**Comment by hanfei1991** [comment]
LGTM

**Comment by hanfei1991** [comment]
@coocood @zimulala PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**